package.json

```diff
@@ -117,7 +117,8 @@
         "start": "node node_modules/react-native/local-cli/cli.js start",
         "compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
         "vscode:prepublish": "gulp",
-        "test": "node ./node_modules/vscode/bin/test"
+        "test": "node ./node_modules/vscode/bin/test",
+        "postinstall" : "node tools/compileTools.js"
```

Comment:
I wouldn't use postinstall. Partly because it is not recommended in general, and partly because it happens at the wrong point in the lifecycle. prepublish might be better... The reason that it worked in postinstall for TACO is that was for the repo itself, which wasn't actually "really" a package, wheras this  repo is.